task8/{UPMessenger}/pom.xml

```diff
@@ -0,0 +1,42 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>{bsu.fpmi.UP}</groupId>
+  <artifactId>{UPMessenger}</artifactId>
```

Comment:
Hey, why you have enclosed your project name in curly braces? I guess, just because you can? ;)
Be careful with this. In Maven `${something}` is the notation to inject prop value into another node (same as ES6 string templates).
So the first thought that came to my head was "what is he going to inject here"? )
Also I doubt that the global Maven repository would accept an artifact named that way, but can't tell it for sure.

Same applies to the folder name, You're definitely seeking an ~adventure for yours~ some very unexpected bug that can only arise in some special conditions -- such bugs are tha hardest to uproot)